spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -425,13 +426,17 @@ def main(argv: Sequence[str]):
                                          param_centerline=param_centerline,
                                          verbose=verbose,
                                          remove_temp_files=arguments.r)
-
     if normalize_pam50:
         fname_vert_level_PAM50 = os.path.join(__data_dir__, 'PAM50', 'template', 'PAM50_levels.nii.gz')
         metrics_PAM50_space = interpolate_metrics(metrics, fname_vert_level_PAM50, fname_vert_level)
+        if verbose == 2:
+            # Get all available vertebral levels from PAM50 template to only include slices from available levels in .csv file
+            levels = get_all_vertebral_level(Image(fname_vert_level_PAM50))
+        else:
+            # Get all available vertebral levels to only include slices from available levels in .csv file
+            levels = get_all_vertebral_level(Image(fname_vert_level))
         metrics = metrics_PAM50_space  # Set metrics to the metrics in PAM50 space to use instead
```

Comment:
If a user specifies `-z` + `-normalize-PAM50`, do they have to specify `-z` in terms of the original image's slices, or the PAM50's slices?

Since we're setting `metrics=metrics_PAM50_space`, I think `-z` has to be in terms of PAM50 slices, and not image slices...

Is this the behavior that we want?

Reply:
I guess it's sort of impossible to keep track of both the original image's slices _and_ the PAM50 slices once normalization is done... :sweat: 

So, there's probably not much we can do here...

Reply:
since this is always perslice, the fact that a user wants to specify -z doesn't have much of an impact, there is no averaging. They will just reduce teh size of the .csv  file. The  behaviour doesn't bother me in that sence!